examples/reserved-ips/versions.tf

```diff
@@ -0,0 +1,16 @@
+##############################################################################
+# Terraform Providers
+##############################################################################
+
+terraform {
+  required_version = ">= 1.3.0"
+  required_providers {
+    ibm = {
+      source = "IBM-Cloud/ibm"
+      # pin above lowest vesion, required for postgresql and IAM auth policy
```

Comment:
this comment is not correct - the version is not pinned, its using a greater than or equals to range, which is probably needed due to other dependencies in the example. Suggest updating or removing the comment